Header header
Header image_header
BoundingBox[] bounding_boxes
string class_name